.github/workflows/pr.yml

```diff
@@ -21,7 +21,7 @@ jobs:
   verify-in-kubernetes:
     name: Verify in Kubernetes
     runs-on: ubuntu-latest
-    needs: build
+#    needs: build
```

Comment:
This change seems unrelated to this pull request.

Reply:
Good catch, exactly, this has nothing to do with this. I will revert